src/swagger-specs/query-service.yaml

```diff
@@ -776,6 +779,287 @@ paths:
         500:
           description: "Internal Server Error"
 
+  /alert-subscriptions:
+    post:
+      operationId: "create_and_subscribe_to_alert"
+      tags:
+        - "Alert-Subscriptions"
+      summary: Creates an alert and subscribes emailIds for a particular assetID
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/x-gw-ims-org-id'
+        - $ref: '#/parameters/x-sandbox-name'
+        - $ref: '#/parameters/x-sandbox-id'
+        - $ref: '#/parameters/x-service-token'
+        - $ref: '#/parameters/x-api-key'
+        - $ref: '#/parameters/x-request-id'
```

Comment:
The description for this header reads: "A unique ID generated by Adobe.io"

Aside from style issues re: "Adobe I/O", if this is an auto-generated header that is added to the request and not provided by the user, we don't need to include it here.